modules/tinymce/src/themes/silver/test/ts/browser/throbber/ThrobberFocusTest.ts

```diff
@@ -0,0 +1,189 @@
+import { FocusTools, UiFinder, Waiter } from '@ephox/agar';
+import { after, before, describe, it } from '@ephox/bedrock-client';
+import { TinyDom, TinyHooks, TinyUiActions } from '@ephox/mcagar';
+import { Focus, Insert, Remove, SelectorFind, SugarBody, SugarDocument, SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+describe('browser.tinymce.themes.silver.throbber.ThrobberFocusTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+  }, [ Theme ], true);
+
+  before(() => {
+    const editor = hook.editor();
+    const input = SugarElement.fromHtml('<input id="tempInput" />');
+    Insert.after(TinyDom.targetElement(editor), input);
+  });
+
+  after(() => {
+    Remove.remove(SelectorFind.descendant(SugarBody.body(), '#tempInput').getOrDie());
+  });
+
+  const openDialog = (editor: Editor) => {
+    return editor.windowManager.open({
+      title: 'Sample Dialog',
+      body: {
+        type: 'panel',
+        items: [
+          {
+            name: 'id',
+            type: 'input',
+            label: 'ID'
+          }
+        ]
+      },
+      buttons: [
+        {
+          type: 'cancel',
+          name: 'cancel',
+          text: 'Cancel'
+        },
+        {
+          type: 'submit',
+          name: 'save',
+          text: 'Save',
+          primary: true
+        }
+      ],
+    });
+  };
+
+  const setProgressState = (editor: Editor, state: boolean, time?: number) => {
+    if (state) {
+      editor.setProgressState(true, time);
+    } else {
+      editor.setProgressState(false);
+    }
+  };
+
+  const pAssertThrobberVisible = async () => {
+    await UiFinder.pWaitForVisible('Wait for throbber to show', SugarBody.body(), '.tox-throbber');
+  };
```

Comment:
```suggestion
  const pAssertThrobberVisible = () =>
    UiFinder.pWaitForVisible('Wait for throbber to show', SugarBody.body(), '.tox-throbber');
```
Remember we prefer single-line lambdas/arrow functions where possible. There's also no need for that async/await here.